tests/backward-compat/upgrade/src/test/groovy/org/apache/bookkeeper/tests/backwardcompat/TestCompatUpgrade.groovy

```diff
@@ -55,13 +55,25 @@ class TestCompatUpgrade {
 
         try {
             def ledger0 = currentRunningBK.createLedger(2, 2,
-                                                        currentRunningCL.digestType("CRC32"),
```

Comment:
can we make digest type `CRC32C` which a default value for bk and other projects like pulsar.

Reply:
@rdhabalia done